src/ioc.config.ts

```diff
@@ -17,6 +18,7 @@ export const Services = {
 	Config: "Config",
 	RuleManager: "RuleManager",
 	RuleEngine: "RuleEngine",
+	DependencyChecker: "DependencyChecker",
```

Comment:
This code was moved to Constants.ts in the `dev` branch. `dev` will be merged to `release` on 9/23. I wonder if it would simpler to branch off of dev for now. This will also affect the imports in RetireJsDepChecker.ts